packages/paste-website/src/pages/patterns/button-vs-anchor/index.mdx

```diff
@@ -152,15 +152,19 @@ Use only one per page; this is the most important action a user needs to take. O
 
 If you decide to place an icon in a button, all button icons should be placed to the left. The only exception to this guidance is when you are moving the user forward (like in a multi-step modal, or when paginating through something). Only then should you use a "**Forward >**" (ChevronRightIcon) button. Don't use an arrow icon, since those are reserved for anchors.
 
-<LivePreview scope={{Button, PlusIcon, Stack}} language="jsx">
+<LivePreview scope={{Button, PlusIcon, ChevronRightIcon, Stack}} language="jsx">
```

Comment:
I think we just need to import ChevronRightIcon to get the website build working. You can do that at the top of this file with the other imports:

`import {ChevronRightIcon} from '@twilio-paste/icons/esm/ChevronRightIcon';`

Reply:
ahhhh, ok that's what I was missing. Thank you!